package.json

```diff
@@ -42,32 +42,28 @@
 		"workerize-loader": "^1.3.0"
 	},
 	"dependencies": {
-		"@babel/core": "^7.10.3",
-		"@babel/plugin-proposal-object-rest-spread": "^7.10.3",
```

Comment:
`plugin-proposal-object-rest-spread` is now included in `preset-env` so I think we should let preset-env do it's job and decide whether or not to include this plugin